pkg/sdk/informer.go

```diff
@@ -18,6 +18,7 @@ import (
 	"context"
 	"time"
 
+	"github.com/operator-framework/operator-sdk/pkg/sdk/metrics"
```

Comment:
Leave a line between 21 and 22. We have the convention of grouping the project related imports separately.